src/test/datascience/widgets/rendererUtils.ts

```diff
@@ -160,3 +161,46 @@ function getOutputCellIndex(outputItem: OutputItem): number | undefined {
         return parseInt(model._vsc_test_cellIndex);
     }
 }
+
+let consoleLoggersHijacked = false;
+/**
+ * Highjack the console log messages & send them to the extension host.
+ * On CI we cannot see the console log messages of the webview, so we need to hijack them.
+ * Sometimes when widgets fail this is very useful in diagnosing what went wrong on the webview side of things.
+ */
+function hijackLogging() {
```

Comment:
Note: This is a test file `src/test/...`, hence this only applies to tests & this function is only called in the widget tests.